jekyll/_cci2/examples-and-guides-overview.adoc

```diff
@@ -0,0 +1,54 @@
+---
+version:
+- Cloud
+- Server v3.x
+---
+= Examples and Guides Overview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Resources for learning CircleCI using examples. 
+:icons: font
+:toc: macro
+:toc-title:
+
+This page provides a collection of examples to help you learn how to build, test, and deploy applications written in the most common programming languages, frameworks, and platforms on CircleCI.
+
+If you are new to CircleCI, we recommend going through our <<getting-started#,Quickstart Guide>> for getting your first successful pipeline.
+
+For basic examples of adding pipeline jobs in a specific execution environment such as a Docker container or Linux VM, review our <<hello-world#,Hello World>> document.
+
+[#languages]
+== Language guides
+
+We’ve created several demo applications in various languages, so you can learn by example from an app written in the same language as your application. Each language listed below has an associated guide and public repository on GitHub.
```

Comment:
```suggestion
We have created demo applications in various languages so you can learn by example. Each language listed below has an associated guide and public repository on GitHub.
```

Reply:
I've taken a lot out of this sentence to feel free to play around, just a suggestion. Only thing that's definitely is removing the contraction "we've" as per style guide. :-D

Reply:
I've approved anyway so feel free to merge when done. Thanks for working on all these language guide tickets! 🍰